### DISCUSSION TITLE
Questions

### DISCUSSION THREAD

**Discussion by AntonioArantesDevscope** [microsoft/PowerPlatformConnectors#1208] [Q&A]
Hello, I was asked to move my certified connector files to a new folder, with a new name. I can't figure out how can I do it because my certified connector with the old name has already been merged.
How can I create a new folder and then put the files inside it if I can even create a folder on this repo? Can someone help me here?

**Comment by ptalrejagithub** [comment]
@AntonioArantesDevscope, can you please synchronize your branch with ours. That is create a folk and fetch and merge the upstream directory https://github.com/microsoft/PowerPlatformConnectors#creating-a-fork. Once you have done the fetch, you can make the changes and then open a new PR.
